dll/win32/windowscodecs/converter.c

```diff
@@ -859,6 +868,7 @@ static HRESULT copypixels_to_32bppBGRA(struct FormatConverter *This, const WICRe
         }
         return S_OK;
     default:
+        FIXME("Unimplemented conversion path!\n");
```

Comment:
Add `source_format`.

Reply:
This is a take over from wine. So i would prefer to keep it.

Reply:
I understand.
NiceToHave: Improve Wine first and/or use `#ifndef __REACTOS__`?

Reply:
Keeping it is better i think. Similar problems like this will be visible in log files in future, not only while testing apps. This should pinpoint us to missing feature.

Reply:
> pinpoint

Isn't that exactly why adding `source_format` value would be useful?